static/css/style.css

```diff
@@ -59,13 +59,12 @@ p, span, a {
 /* Buttons */
 
 .btn {
-  font-family: Poppins;
+  font-family: Poppins, sans-serif;
```

Comment:
It seems that this PR contains a lot of different changes that could be separate.

Reply:
removed style changes